pegen/grammar_visualizer.py

```diff
@@ -0,0 +1,92 @@
+import argparse
```

Comment:
This is a nice tool. A nomenclature nit: It doesn't visualize the grammar but the AST (for the grammar). And that's useful separately.